### PR TITLE
Add URL to the configuration options and bump version

### PR DESCRIPTION
**Pull request by kbacha**
Using a URL in an environment variable is much easier than parsing out the entire scheme into options. However, it's easy to maintain backwards compatibility and just give preference to the URL.

### CODE REVIEW

File: weedmaps_logger.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.11'
+  spec.add_development_dependency 'bundler', '~> 1.9'
```
Comment (jcraigk) [inline weedmaps_logger.gemspec]: why downgrade bundler?

Comment (kbacha) [inline weedmaps_logger.gemspec]: circleci uses 1.9.5

Comment (kbacha) [inline weedmaps_logger.gemspec]: I'll add a comment.

File: lib/wm/logger/rabbitmq_client.rb [hunk 2]
```diff
@@ -3,13 +3,17 @@
...
+              else
+                Bunny.new(
+                  host: config[:host] || '127.0.0.1',
+                  port: config[:port] || 5672,
+                  user: config[:user],
+                  pass: config[:pass],
+                  vhost: config[:vhost]
+                )
+              end
       @exchange_type = config[:exchange_type] # must be 'direct' or 'fanout'
```
Comment (jcraigk) [inline lib/wm/logger/rabbitmq_client.rb]: I prefer the parentheses and at least my version/config of rubocop doesn't complain.  I'm not a big fan of indented blocks like this where indentation length is not scope-based but based on the length of the variable that we're assigning to or whatever.  I'm fine with either in this case cuz it's simple/small in general.

**Comment by jcraigk** [comment]
LGTM

File: lib/wm/logger/rabbitmq_client.rb (hunk 2 above)
Comment (kbacha) [inline lib/wm/logger/rabbitmq_client.rb]: If not explicitly stated I tend to use the bbatsov style guide which:
https://github.com/bbatsov/ruby-style-guide#indent-conditional-assignment

If the line were long I would switch it to be only indented once.

**Pull request merged**